include/libbase64.h

```diff
@@ -3,6 +3,39 @@
 
 #include <stddef.h>	/* size_t */
 
+
+#if defined(_WIN32) || defined(__CYGWIN__)
+#define BASE64_SYMBOL_IMPORT __declspec(dllimport)
+#define BASE64_SYMBOL_EXPORT __declspec(dllexport)
+#define BASE64_SYMBOL_PRIVATE
+
+#elif __GNUC__ >= 4
+#define BASE64_SYMBOL_IMPORT   __attribute__ ((visibility ("default")))
```

Comment:
I think import symbols don't need this.
```suggestion
#define BASE64_SYMBOL_IMPORT
```

Reply:
I'm not an expert w.r.t. gcc symbol visibility; I looked at the [`GenerateExportHeader`](https://cmake.org/cmake/help/latest/module/GenerateExportHeader.html) output and noticed that they do define the import macro with said attribute (s. [`GenereateExportHeader.cmake:310`](https://gitlab.kitware.com/cmake/cmake/-/blob/master/Modules/GenerateExportHeader.cmake#L310)). However, I don't know the rationale.

Reply:
My interpretation is that adding `visibility("default")`, you're instrumenting the compiler to export a symbol.
So adding this as a consumer will cause you to re-export the symbol.